AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -171,8 +176,58 @@ - (void)didLoad
           _placeholderImageNode.frame = self.bounds;
           [self insertSubnode:_placeholderImageNode atIndex:0];
         });
-      }];
-    });
+      }
+    }];
+  });
+}
+
+- (void)interfaceStateDidChange:(ASInterfaceState)newState fromState:(ASInterfaceState)oldState
+{
+  [super interfaceStateDidChange:newState fromState:oldState];
+	
+	BOOL nowVisible = ASInterfaceStateIncludesVisible(newState);
+	BOOL wasVisible = ASInterfaceStateIncludesVisible(oldState);
+
+  if (!nowVisible) {
+    if (wasVisible) {
+      if (_shouldBePlaying) {
+        [self pause];
+        _shouldBePlaying = YES;
```

Comment:
@appleguy This isn't one of my additions, but by the looks of things it's intended to pause and play videos that are set to 'auto play' as they come in and out of view. 

Running this with the debugger on the VideoTableView example this does appear to be working as expected, so it doesn't seem to be a problem.

The `_shouldBePlaying = YES` is redundant however and I've removed this.